applications/DEMApplication/custom_conditions/RigidFace.cpp

```diff
@@ -343,7 +343,8 @@ void RigidFace3D::ComputeConditionRelativeData(int rigid_neighbour_index,
     if (points == 3 || points == 4)
     {
         unsigned int dummy_current_edge_index;
-        contact_exists = GeometryFunctions::FacetCheck(this->GetGeometry(), node_coordinates, radius, LocalCoordSystem, DistPToB, TempWeight, dummy_current_edge_index);
+        bool dummy_inside;
```

Comment:
No need to call it _dummy_. It's ok if we don't use it after calling the function. It's something usual if the function requires the argument.

Reply:
Changed variable name.